src/Statistics/HistogramTest.cpp

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <array>
+#include <cstdint>
+#include <numeric>
+#include <vector>
+
+#include "HistogramPrivate.h"
+#include "Statistics/DataSet.h"
+#include "Statistics/Histogram.h"
+
+namespace orbit_statistics {
+
+constexpr uint64_t kBandwidth = 5;
+constexpr size_t kDataSetSize = 8;
+constexpr std::array<uint64_t, kDataSetSize> kRawDataSet{11ULL, 12ULL, 14ULL,  18ULL,
+                                                         19ULL, 30ULL, 100ULL, 58ULL};
+constexpr uint64_t kMin = *std::min_element(kRawDataSet.begin(), kRawDataSet.end());
+constexpr uint64_t kMax = *std::max_element(kRawDataSet.begin(), kRawDataSet.end());
+
+const std::vector<uint64_t> raw_data_set(kRawDataSet.begin(), kRawDataSet.end());
+
```

Comment:
I think there should be more attention to the naming of the first part of the names of this test. This is used to group tests together. Ideally, it should all be `HistogramTest`. It doesn't necessarily have to be but there's too much variation, e.g., `HistogramTest` and `HistogramBuildTest`.